CHANGES.rst

```diff
@@ -127,6 +127,8 @@ Internal Changes
   :user:`omegak`)
 - Add ``before-registration-actions`` template hook (:issue:`4524`, thanks
   :user:`omegak`)
+- Improve ``IndicoDateField`` rendering and validators (:issue:`4535`, thanks
+  :user:`omegak`)
```

Comment:
hm not sure if it's useful... it's not really relevant for end users nor plugin developers

Reply:
It would still be relevant to specify that `DateRange ` and `LinkedDate` validators have been added.